<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detailDialog</class>
 <widget class="QDialog" name="detailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Segoe UI&quot;;

background-color: qlineargradient(spread:reflect, x1:0.095, y1:0.176, x2:0.391, y2:0.471318, stop:1 rgba(11, 11, 11, 248));</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>621</width>
     <height>441</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_state">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>state: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_restaurant">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(0, 0, 0, 0));</string>
    </property>
    <property name="text">
     <string>restauran:     </string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>611</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>-70</y>
      <width>671</width>
      <height>511</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/img/pexels-negative-space-34650.jpg);
border -radius:20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_state</zorder>
   <zorder>label_restaurant</zorder>
   <zorder>tableView</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
